api/web_hub.go

```diff
@@ -60,7 +60,7 @@ func (h *Hub) Stop() {
 	h.stop <- "all"
 }
 
-func (h *Hub) Start() {
+func (h *Hub) StartAndForget() {
```

Comment:
Can we also leave all the `Start` functions the way they were? Sorry, I should have mentioned those as a special case before